components/library/jsoncpp/Makefile

```diff
@@ -0,0 +1,39 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 erwinlem
+#
+
+BUILD_STYLE=cmake
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		jsoncpp
+COMPONENT_VERSION=	1.9.5
+COMPONENT_SUMMARY=	A C++ library for interacting with JSON.
+COMPONENT_PROJECT_URL=	https://github.com/open-source-parsers/jsoncpp
+COMPONENT_FMRI=		library/jsoncpp	
+COMPONENT_CLASSIFICATION=
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL=	https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:f409856e5920c18d0c2fb85276e24ee607d2a09b5e7d5f0a371368903c275da2
+COMPONENT_LICENSE=	MIT
+COMPONENT_LICENSE_FILE=	LICENSE
+
+
+TEST_TARGET=$(NO_TESTS) # if no testsuite enabled
+include $(WS_MAKE_RULES)/common.mk
```

Comment:
You don't need to disable tests. They work fine. You could additionally provide expected test results and add filters to normalize (e.g. filter out timings) them.

Reply:
I just copied the template, but it actually passes. Updated the test & made a filter.